X86/X86MachineInstructionRaiserUtils.cpp

```diff
@@ -455,9 +455,10 @@ const MachineInstr *X86MachineInstructionRaiser::getPhysRegDefiningInstInBlock(
 
     // If the instruction has a define
     if (MI.getNumDefs() > 0) {
-      for (auto MO : MI.defs()) {
+      // MI.defs() does not include implicit defs
+      for (auto MO : MI.operands()) {
```

Comment:
I'm not sure if there's a better way to do this without looping over all operands. I think this might have a performance impact, but I'm not sure how high it is.